shorty/autoconfig.py

```diff
@@ -10,4 +10,8 @@
         'nuit',
     ],
     'NUIT_GLOBAL_TITLE': 'Shorty',
+    'CSP_SCRIPT_SRC': (
+        "'self'",
+        "https://cdnjs.cloudflare.com",
```

Comment:
Done. Please merge if you have no further comments!

On 25 July 2016 at 16:40, Mike Bryant <notifications@github.com> wrote:

> In shorty/autoconfig.py
> <https://github.com/ocadotechnology/djshorty/pull/21#discussion_r72088118>
> :
>
> > @@ -10,4 +10,8 @@
> >          'nuit',
> >      ],
> >      'NUIT_GLOBAL_TITLE': 'Shorty',
> > +    'CSP_SCRIPT_SRC': (
> > +        "'self'",
> > +        "https://cdnjs.cloudflare.com",
>
> Do we trust that *any* library on the cdnjs site is trustworthy? We
> probably want to whitelist the specific libraries we're interested in
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ocadotechnology/djshorty/pull/21/files/56b03470be25cf0a10e8a4cef130c5df457bf6c3#r72088118>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA13JdmAr3jbbHXl8d7pakD2Qo2HqmJRks5qZNjngaJpZM4JUF21>
> .
>